wgpu-core/src/command/render.rs

```diff
@@ -896,7 +938,9 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         .bind_groups
                         .use_extend(&*bind_group_guard, bind_group_id, (), ())
                         .unwrap();
-                    bind_group.validate_dynamic_bindings(offsets).unwrap();
+                    bind_group
+                        .validate_dynamic_bindings(offsets)
+                        .map_err(RenderPassError::from)?;
```

Comment:
similarly here, we don't need this `map_err()`, just do `validate_dynamic_bindings(offsets)?`